packages/vue-form-validator/src/form-validator/utils/is-valid-url.ts

```diff
@@ -1,8 +1,10 @@
+/* global URL */
```

Comment:
imo it's better to have it defined in eslint.config or as suggested before maybe it's worth using `globals` package in the base eslint config to cover all possible cases